src/pages/Delivery/index.jsx

```diff
@@ -26,6 +27,7 @@ const Delivery = () => {
 
   const toChat = () => history.push(`/chat/${id}`)
 
+  // @ts-ignore
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142056460-e70ad01a-5fe9-415f-859a-ef61a4b6920b.png)

Reply:
This is optional, we can add proper typing on JSDocs

@Katsu08 please help me adding a ticket low priority to solve this one